### DISCUSSION TITLE
Connecting WD via Beeline

### DISCUSSION THREAD

**Discussion by sivaponting** [ExpediaGroup/waggle-dance#286] [General]
Hi Team,
Is thr any way to connect the WD using beeline? 
WD listening on thrift port
Beeline needs jdbc

**Comment by patduin** [comment]
I suspect you need to run HiveServer2 for that. This is probably best asked/search in the Apache Hive docs. If you can connect to Hive thrift uri via Beeline it should work in the same way with WD.

**Comment by sivaponting** [comment]
> I suspect you need to run HiveServer2 for that. This is probably best asked/search in the Apache Hive docs. If you can connect to Hive thrift uri via Beeline it should work in the same way with WD.

Here is my use case:
Metastore running on 3 machines
poc11a, poc12a & poc13a

WD running on poc11a
HiveServer2 running on poc12a

How to running beeline in this case.
Does the beeline accept thrift url? Here we are using jdbc url only.

**Comment by patduin** [comment]
I haven't used beeline much, so can't help there. I usually just use the
hive-cli or spark-sql.

Op wo 16 aug 2023 om 12:31 schreef Sivakumar Duraimanickam <
***@***.***>:

**Comment by patduin** [comment]
Going to close this as this is not a Waggle Dance issue.
